examples/src/test/scala/docs/kafka/KafkaDocExample.scala

```diff
@@ -130,11 +225,10 @@ object KafkaDocExample {
    * {{{
    * bin/kafka-topics.sh --create --bootstrap-server localhost:9092 --replication-factor 1 --partitions 3 --topic words
    * bin/kafka-console-producer.sh --bootstrap-server localhost:9092 --topic words
```

Comment:
Curious why we're not using Alpakka Kafka for topic admin and the producer and consumer?

Reply:
Probably just because I don't know enough about it, but also nice to see that there is no magic going by using it bare bones.

Reply:
For docs purposes it's fine.  We can update it later if it makes sense.

Reply:
to be clear, these comments about bin/kafka are not shown in docs, it was only for my own convenience when trying it out